hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/util/TestCombinedHostsFileReader.java

```diff
@@ -44,8 +52,12 @@ public class TestCombinedHostsFileReader {
   private final File legacyFile =
       new File(TESTCACHEDATADIR, "legacy.dfs.hosts.json");
 
+  @Mock
+  private Callable<DatanodeAdminProperties[]> callable = Mockito.mock(Callable.class);
```

Comment:
I don't think you need to call the mock method explicitly if you are using the mock annotation and initMocks. This will resolve the unchecked conversion.